style.css

```diff
@@ -0,0 +1,245 @@
+	color: white;
+	font-size: 2rem;
+	display: flex;
+	justify-content: center;
+}
+
+div.about-catcher{
+    width: 100%;
+ 	padding: 10px;
+    background-color: #3C0051;
+}
+
+section.about{
+	    width: 100%;
+	    height: 800px;
+	    background-color: #3C0051;
+	    margin-top: 30px;
+	    display: flex;
+	    flex-wrap: wrap;
+	    flex-direction: column;
+
+}
+
+.about-img{
+	width:600px;
+	height:600px;
+	object-fit:contain;	
+	margin-right: 10px;
+}
+
+h2.about{
+	margin-left:15px;
+	color: white;
+	font-size: 2rem;
+	text-decoration: underline;	
+}
+
+p.about{
+	width:60%;
+	color: white;
+	font-size: 1.5rem;
+	margin-left:15px;
+}
+
+div.portfolio-catcher {
+    width: 100%;
+ 	padding: 10px;
+    background-color: #453952;
+	margin-top:50px;
+}
+
+section.portfolio-intro { width: 100%;
+    height: 150px;
+  	background-color:#453952;
+	color: white;	
+}
+
+h2.portfolio-intro {
+	margin-left:15px;
+	text-decoration: underline;
+}
+
+p.portfolio-intro {
+	margin-left:15px;
+	color: white;
+	font-size: 1rem;
+}
+
+section.portfolio {
+	display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+    justify-content: center;
+    align-content: center;
+	color: white;
+}
+
+
+div.portfolio-piece1 {
+	background-color: #3e2182;
+	margin-top: 30px;
+	border: 5px solid black;
+	width: 70%;
+	height: 600px;
+	}
+
+div.portfolio-piece2 {
+	background-color: #3e2182;
+	margin-top: 30px;
+	border: 5px solid black;
+	width: 70%;
+	height: 600px;
+	}
+
+div.portfolio-piece3 {
+	background-color: #3e2182;
+	margin-top: 30px;
+	border: 5px solid black;
+	width: 70%;
+	height: 600px;
+	}
+
+div.portfolio-piece4 {
+	background-color: #3e2182;
+	margin-top: 30px;
+	border: 5px solid black;
+	width: 70%;
+	height: 600px;
+	}
+
+div.portfolio-piece5 {
+		background-color: #3e2182;
+		margin-top: 30px;
+		border: 5px solid black;
+		width: 70%;
+		height: 600px;
+	}
+
+.portfolio-text-head{
+	text-decoration: underline;	
+	}
+	
+.container {
+	display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+    align-content: stretch;
+    align-items: center;
+}	
+
+.portfolio-image{
+	object-fit: contain ;
+		width:70%;
+		height:400px;
+	}
+
+
+section.contact{
+	margin-top:30px;
+	padding-top: 10px;
+    width: 100%;
+    height: 150px;
+  	background-color:#322245;	
+}
+
+h2.contact {
+	margin-left:15px;
+	color: white;
+	text-decoration: underline;
+}
+p.contact{
+	color: white;
+}
+
+a.contact{
+	text-decoration: none;
+	color: white;
+}
+
+p.contact-intro{
+	color: white;
+	font-size: 1rem;
+	font-weight: bold;
+}
+
+section.contact-tidy{
+    margin-top: -20px;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-content: stretch;
+    justify-content: space-between;
+	margin-left:20px;
+	margin-right:15px;
+}
+
+@media screen and (max-width: 1000px) {
+	section.contact{
+   width: 100%;
+    height: 300px;
+  	background-color:#322245;
+	color: white;}
+	section.about{
+	 width: 100%;
+	 height: 2000px; 
```

Comment:
once again, we should not be setting the height of this element, rather it should be left to be determined by its content